x/authority/client/cli/tx_update_chain_info.go

```diff
@@ -0,0 +1,45 @@
+package cli
+
+import (
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+
+	"github.com/zeta-chain/zetacore/x/authority/types"
+)
+
+func CmdUpdateChainInfo() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-chain-info [chain-info-json-file]",
+		Short: "Update the chain info",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			// Read the chain info from the file using os package and unmarshal it into the chain info variable
+			var chainInfo types.ChainInfo
+			chainInfoBytes, err := os.ReadFile(args[0])
+			if err != nil {
+				return err
+			}
+			if err := chainInfo.Unmarshal(chainInfoBytes); err != nil {
+				return err
+			}
```

Comment:
Can we move this to a separate function, `ParseChainInfo`, and have a small test case for that? The JSON data can be  stored under the sample directory

Reply:
[14d06c3](https://github.com/zeta-chain/node/pull/2287/commits/14d06c3023f05643a371bef2967570693b29aa7d)